pkg/apis/eventing/register.go

```diff
@@ -56,6 +56,10 @@ const (
 	// ScopeCluster indicates the resource must be
 	// handled by the cluster-scoped component
 	ScopeCluster = "cluster"
+
+	// EventTypesAnnotationKey is the annotation key to specify
+	// if a Source has event types defines in its CRD.
+	EventTypesAnnotationKey = "registry.knative.dev/eventTypes"
```

Comment:
Just use the source duck type annotation? why a new special annotation?

Reply:
this is needed to be able to get other info from this registry annotation, such as description and schema, in order to create a more complete eventtype

Reply:
I would still use the source annotation and test to see if the registry info is there

we risk annotation burden with too many. The bar gets higher and higher to make a source if we keep adding these annotations

Reply:
OOOH

Sorry. The registry is _in_ the annotation, ignore me.